module/parser/src/commonMain/kotlin/org/cru/godtools/shared/tool/parser/ParserConfig.kt

```diff
@@ -16,6 +18,7 @@ data class ParserConfig private constructor(
     private val supportedFeatures: Set<String> = emptySet(),
     internal val parsePages: Boolean = true,
     internal val parseTips: Boolean = true,
+    internal val parserDispatcher: CoroutineDispatcher = Dispatchers.Default,
```

Comment:
@wrandall22 the type of `parserDispatcher` is defined as `CoroutineDispatcher`, which indicates that it's non-null.

It would be nullable if it was `CoroutinesDispatcher?` instead, and then we would be forced to deal with the null case when necessary otherwise the code wouldn't compile.